config/initializers/active_record_extensions.rb

```diff
@@ -4,3 +4,38 @@ class ActiveRecord::Base
   include StripWhitespace
   include Parameterizable::ById
 end
+
+# Fixes the issue with custom counter cache column names
```

Comment:
interesting. I just commented out those two monkey patches, ran tests and all is well. I guess those two can go.